src/config/env.js

```diff
@@ -1,44 +1,28 @@
-export let kitsuConfig = {};
+export const kitsuConfig = {
+  authConfig: {
+    CLIENT_ID: 'dd031b32d2f56c990b1425efe6c42ad847e7fe3ab46bf1299f05ecd856bdb7dd',
+    CLIENT_SECRET: '54d7307928f63414defd96399fc31ba847961ceaecef3a5fd93144e960c0e151',
```

Comment:
these are okay to be left in, they're public credentials that a lot of api clients use right now. once we have dedicate creds for the app we can hide them.